### PR TITLE
User-story/2342-clean-up-padding-in-inner-frames

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#2342

### CODE REVIEW

File: OTAnalytics/plugin_ui/customtkinter_gui/custom_containers.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        text_color = self._segmented_button.cget("text_color")
+        self.configure(text_color_disabled=text_color)
+        self._segmented_button.configure(state="disabled")
+
+    # def _override_padding_above(self) -> None:
+    #     """
+    #     Overrides the minimum size of the dummy row above the tabview to prevent
+    #     padding.
+    #     """
+    #     self.grid_rowconfigure(0, weight=0, minsize=0)
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/custom_containers.py:72]: Could this code be removed?

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_project.py [hunk 2]
```diff
@@ -3,23 +3,49 @@
...
-class FrameProject(AbstractFrameProject, CTkFrame):
+class TabviewProject(CustomCTkTabview):
+    def __init__(
+        self,
+        viewmodel: ViewModel,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+        self._viewmodel = viewmodel
+        self.PROJECT: str = "Project"
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/frame_project.py:30]: `self.PROJECT` looks more like `self.title`

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_tracks.py [hunk 3]
```diff
@@ -20,27 +22,14 @@ def introduce_to_viewmodel(self) -> None:
         self._viewmodel.set_tracks_frame(self)
 
     def _get_widgets(self) -> None:
-        self.label = CTkLabel(master=self, text="Tracks")
+        # self.label = CTkLabel(master=self, text="Tracks")
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/frame_tracks.py:25]: Could this code be removed?

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_videos.py [hunk 4]
```diff
@@ -36,16 +37,16 @@ def _get_widgets(self) -> None:
...
             master=self, text="Remove", command=self._viewmodel.remove_videos
         )
 
     def _place_widgets(self) -> None:
-        self.label.grid(
-            row=0, column=0, columnspan=2, padx=PADX, pady=PADY, sticky=STICKY
-        )
+        # self.label.grid(
+        #     row=0, column=0, columnspan=2, padx=PADX, pady=PADY, sticky=STICKY
+        # )
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/frame_videos.py:49]: Could this code be removed?

File: OTAnalytics/plugin_ui/customtkinter_gui/gui.py [hunk 5]
```diff
@@ -179,14 +176,19 @@ def _show_gui(self) -> None:
...
     def _get_widgets(self) -> None:
-        self._app.grid_columnconfigure(0, minsize=350, weight=0)
-        self._app.grid_columnconfigure(1, weight=1)
-        self._app.grid_rowconfigure(0, weight=1)
-        self._navigation = FrameNavigation(master=self._app, viewmodel=self._viewmodel)
+        self._navigation = FrameNavigation(
+            master=self._app,
+            viewmodel=self._viewmodel,
+            width=336,
+            # fg_color=self._app.cget("fg_color"),
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/gui.py:183]: Could this code be removed?

File: OTAnalytics/plugin_ui/customtkinter_gui/gui.py [hunk 6]
```diff
@@ -179,14 +176,19 @@ def _show_gui(self) -> None:
...
             master=self._app, viewmodel=self._viewmodel, layers=self._layers
         )
 
     def _place_widgets(self) -> None:
+        self._app.grid_columnconfigure(0, minsize=300, weight=0)
+        self._app.grid_columnconfigure(1, weight=1)
+        self._app.grid_rowconfigure(0, weight=1)
         self._navigation.grid(row=0, column=0, padx=PADX, pady=PADY, sticky=STICKY)
-        self._content.grid(row=0, column=1, padx=PADX, pady=PADY, sticky=STICKY)
+        self._content.grid(row=0, column=1, padx=PADX, pady=PADY + 13, sticky=STICKY)
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/gui.py:194]: 13 as magic number might be bad?
Can we extract it into a constant?

**Review by briemla — CHANGES_REQUESTED** [review]

**Comment by martinbaerwolff** [comment]
Also had a look at the whole new design on windows, everything looks fine

**Review by briemla — APPROVED** [review]

**Pull request merged**